cpp/rss_aggregator.cc

```diff
@@ -46,7 +46,7 @@ namespace {
 
 [[noreturn]] void Usage() {
     ::Usage(
-        "[--use-web-proxy] subsystem_type email_address xml_output_path\n"
+        "optional:[--use-web-proxy] subsystem_type email_address xml_output_path\n"
```

Comment:
Whether or not to dowload and whether or not to use the web proxy when downloading are 2 different things and should be expressed within 2 different parameters